app/domain/authentication/authn/authenticator.rb

```diff
@@ -1,40 +1,54 @@
 # frozen_string_literal: true
 
+require 'command_class'
+
 # default conjur authenticator
-#
 module Authentication
   module Authn
+    Authenticator = CommandClass.new(
+          dependencies: {
+            role_cls:  ::Role,
+            credentials_cls: ::Credentials
+          },
+          inputs:       [:authenticator_input]
+      ) do
+
+      extend Forwardable
+      def_delegators :@authenticator_input, :account, :credentials, :username
+
+      def call
+        return false unless role_credentials
```

Comment:
in other authenticators we do not return false. we either let the command class finish its work or raise an error if something happened.

Reply:
I had to do it because tests did not pass otherwise

Reply:
i see. thanks for clarifying.